bin/main.dart

```diff
@@ -452,20 +456,15 @@ Future<Map<String, String>> _loadStrings(String lang, String dirPath) async {
   var localizedStrings = <String, String>{};
   try {
     final file = File('$dirPath/$lang.json');
-    if (!await file.exists()) {
-      return localizedStrings;
-    }
+    if (!await file.exists()) return localizedStrings;
     final jsonString = await file.readAsString();
-    if (jsonString.isEmpty) {
-      return localizedStrings;
-    }
+    if (jsonString.isEmpty) return localizedStrings;
```

Comment:
please return bracket, it is a rule from linter

Reply:
Ok